recipes/sqlite3/all/CMakeLists.txt

```diff
@@ -150,7 +150,6 @@ install(DIRECTORY source_subfolder/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
 if(SQLITE3_BUILD_EXECUTABLE)
     add_executable(sqlite3-bin source_subfolder/shell.c)
     target_link_libraries(sqlite3-bin PRIVATE ${PROJECT_NAME})
-    set_target_properties(sqlite3-bin PROPERTIES OUTPUT_NAME "sqlite3")
```

Comment:
don't do that, sqlite3 executable name must be sqlite3.

Reply:
Agreed, that's the original name, you must not change it.

Reply:
I think this was already addressed in https://github.com/conan-io/conan-center-index/pull/8586/commits/3170293346757f6452044fe30523807f5af11c2e

Reply:
Right, removing the block.